src/App.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './App.css';
+import {Profile} from './profile/profile'
+
+
+function App() {
+  const styleObject={ border:" 2px solid black" ,};
+  function handleName(name){
+      alert(name);
+  }
+  return (
+    <>
+    <Profile style={styleObject} fullName="Andrew" bio="" profession="" alertName={handleName} >
+      <img  style={styleObject} src="images/img1.png" alt="profileImage" />
+      <br></br>
```

Comment:
`br` have no children, so they can be written as self-closing in JSX:

```js
<br />
```

Also, `br` elements should be used for line breaks, which is not the case here. If you want to visually separate the 2 images, use CSS.